File: benchmark/python/geometric_series.py
import time
start = time.time() * 1000
r = 0.999
s = 0.0
term = 1.0
for _ in range(1000000):
    s += term
    term *= r
end = time.time() * 1000
print(f"Sum: {s}")
print(f"Time: {end - start:.0f}ms")

File: benchmark/php/geometric_series.php
<?php
$r = 0.999; $sum = 0; $term = 1;
$start = hrtime(true);
for ($i = 0; $i < 1000000; $i++) { $sum += $term; $term *= $r; }
$end = hrtime(true);
$ms = ($end - $start) / 1e6;
echo "Sum: " . $sum . "
";
echo "Time: " . round($ms) . "ms
";
